### PR TITLE
Portfolio-project: Popup window

### PR DESCRIPTION
**Pull request by theodoku**
1. Created a branch
2. Created a modal-popup section in index.html
3.  use a JavaScript array to store all of the information for all projects
4. Update the main page so that the projects section is created dynamically using the information stored in  JavaScript object.
5. Styled Desktop popup using media query

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,7 +97,7 @@ <h2 class="connect-headline"><a href="#">LET’S CONNECT</a></h2>
 
     <!-- Works Section -->
     <section class="works" id="portfolio">
-        <div class="cards">
+        <!-- <div class="cards">
             <div class="project-image">
                 <img src="images/cardImg1.png" class="portfolio-pic"   alt="portfolio picture">
             </div>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure you remove all the commented code under this work section from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

File: style.css [hunk 2]
```diff
@@ -599,6 +660,117 @@ button {
...
+  width: 85%;
+  margin: 10px auto;
+  padding: 0 16px;
+  border-radius: 16px;
+  overflow: auto;
+  padding-top: 2%;
+  padding-bottom: 4%;
+}
+
+.modal_wrapper {
```
Comment (Owusu-Desmond) [inline style.css]: - [ ] Please make sure that you adjust the style of your popup modal to match the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=EZxwm3g8u4jKVfW8-0). comparing both designs from the screenshot below,
  - the Figma design popup background is blurred but yours is not
  - The margin of your popup does not match the Figma design 
  
  ### your design
  
Screenshot from 2023-01-31 18-52-51

### Figma Design
Screenshot couldn't upload so check the [figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=EZxwm3g8u4jKVfW8-0)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @supreeti and @theodoku , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @theodoku, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**